contracts/ToucanRegenBridge.sol

```diff
@@ -31,6 +35,23 @@ contract ToucanRegenBridge is Ownable, Pausable {
     // event emited when we bridge tokens back from Regen Ledger and issue on TCO2 contract
     event Issue(string sender, address recipient, address tco2, uint256 amount);
 
+    // ----------------------------------------
+    //      Modifiers
+    // ----------------------------------------
+
+    modifier isRegenAddress(bytes calldata account) {
+        // verification: checking if account starts with "regen1"
+        require(account.length >= 44, "regen address is at least 44 characters long");
+        bytes memory prefix = "regen1";
+        for (uint8 i = 0; i < 6; ++i)
+            require(prefix[i] == account[i], "regen address must start with 'regen1'");
+        _;
```

Comment:
just curious, what does this line do?

Reply:
The rest of the function code that is modified is executed.

Reply:
https://www.tutorialspoint.com/solidity/solidity_function_modifiers.htm